src/announce/announce.js

```diff
@@ -0,0 +1,106 @@
+import {LitElement, html} from 'lit-element/lit-element';
+
+import {readExpiringLocalValue, setExpiringLocalValue} from '../_helpers/storageUtility';
+
+const primo_specific_padding = html`
+  <style type="text/css">
+    /* site-specific code for primo's announce-banner.js - adds padding below the search bar */ 
+    prm-search-bar { padding-bottom: var(--primo-announce-padding, 60px); }
+    .__xs prm-search-bar, .__sm prm-search-bar { padding-bottom: var(--primo-announce-padding-small, 68px); }
+  </style>
+`;
+
+/**
+ * display time-sensitive message, call-to-action across the top of a screen
+ */
+export default class BULAnnounce extends LitElement {
+
+  // don't need 'slot' functionality, so lets use Light DOM
+  createRenderRoot(){ return this; }
```

Comment:
explain the non-arbitrariness of this decision. if this used shadow DOM, it wouldn't be styled correctly